justfile

```diff
@@ -12,17 +12,31 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-# Note:
-#
-# This Makefile is work in progress. Currently it contains only a small subset
-# of all steps.
+all: check-test-images check-licenses test stest build-release
+
+build:
+    cargo build
+
+build-release:
+    cargo build --release
 
-all: check-test-images check-licenses
+clean:
+    cargo clean
+    ./tools/dev_scripts/ankaios-clean
 
 check-licenses:
-	cargo deny check licenses
+    cargo deny check licenses
 
 # Prevent non ghcr.io images to be used in test due to rate limit problem
 check-test-images:
-	test -z "$(shell find tests/resources/configs -type f -exec grep -H -P 'image: (?!ghcr\.io/|image_typo:latest)' {} \;)"
+    test -z "$(find tests/resources/configs -type f -exec grep -H -P 'image: (?!ghcr\.io/|image_typo:latest)' {} \;)"
+
+test:
+    cargo test
```

Comment:
In CI build we use `cargo nextest run`. Shouldn't we use the same here?

Reply:
```suggestion
    cargo nextest run
```